assets/js/components/indicators.js

```diff
@@ -11,6 +11,17 @@ const indicatorMetricClass = {
   "": ".indicator-metric--no-status",
 };
 
+function insertSpaces(array) {
+  let result = [];
+  array.forEach((o, i, a) => {
+    result.push(o);
+    if (i !== a.length) {
```

Comment:
what are o, i and a? I don't really understand this

Reply:
is this an array zip or join with nbsp's? that might be simpler

Reply:
I've improved the code of this function, generalised it into a utility and added some documentation.